bigtable/client/filters_test.cc

```diff
@@ -0,0 +1,76 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "bigtable/client/filters.h"
+
+#include <gmock/gmock.h>
+
+#include "bigtable/client/chrono_literals.h"
+
+/// @test Verify that functions to create simple filters work as expected.
+TEST(FiltersTest, Simple) {
+  namespace btproto = ::google::bigtable::v2;
```

Comment:
Should this be at the top of the file so that you don't have to duplicate this for every test?